unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/FriendsController/Tests/FriendsControllerShould.cs

```diff
@@ -338,6 +338,63 @@ public void AllocateByUserIdWhenReceiveFriendRequest()
             Assert.AreEqual("hey!", request.MessageBody);
         }
 
+        [Test]
+        public void AllocateByUserIdWhenReceiveFriendRequestWithSocialClient()
+        {
+            GameObject go = new GameObject();
+            var component = go.AddComponent<MatrixInitializationBridge>();
```

Comment:
i think you should initialize this field into the `SetUp` method to avoid creating it over and over in each test